src/Text/Parser/Char.hs

```diff
@@ -367,3 +368,7 @@ instance CharParsing ReadP.ReadP where
   notChar c = ReadP.satisfy (/= c)
   anyChar   = ReadP.get
   string    = ReadP.string
+
+instance CharParsing ReadPrec.ReadPrec where
+  satisfy = ReadPrec.lift . satisfy
+  anyChar = ReadPrec.get
```

Comment:
Why not implement `char`, `notChar`, and `string` as the `CharParsing ReadP` instance does?